diff --git a/cmd/derper/cert.go b/cmd/derper/cert.go
index db84aa515..7a3e1351c 100644
--- a/cmd/derper/cert.go
+++ b/cmd/derper/cert.go
@@ -88,9 +88,9 @@ func (m *manualCertManager) TLSConfig() *tls.Config {
 }
 
 func (m *manualCertManager) getCertificate(hi *tls.ClientHelloInfo) (*tls.Certificate, error) {
-	if hi.ServerName != m.hostname {
-		return nil, fmt.Errorf("cert mismatch with hostname: %q", hi.ServerName)
-	}
+	//if hi.ServerName != m.hostname {
+	//	return nil, fmt.Errorf("cert mismatch with hostname: %q", hi.ServerName)
+	//}
 
 	// Return a shallow copy of the cert so the caller can append to its
 	// Certificate field.

